homeassistant/components/venstar/__init__.py

```diff
@@ -89,5 +89,5 @@ def device_info(self) -> DeviceInfo:
             name=self._client.name,
             manufacturer="Venstar",
             model=f"{self._client.model}-{self._client.get_type()}",
-            sw_version="{}.{}".format(*(self._client.get_firmware_ver())),
+            sw_version=f"{self._client.get_firmware_ver()[0]}.{self._client.get_firmware_ver()[1]}",
```

Comment:
Please just make one call to `self._client.get_firmware`. Store the result in a local variable first.

Reply:
Done